core/proguard-consumer-rules.pro

```diff
@@ -56,5 +51,8 @@
 -keepclassmembers class com.rudderstack.android.sdk.core.RudderTraits { java.util.Map extras; }
 
 # Required for DBEncryption feature using SQLCipher
--keep class net.sqlcipher.** { *; }
--keep class net.sqlcipher.database.* { *; }
```

Comment:
Same with this. Change the package name

Reply:
Based on our internal discussion, we identified that rules are already added by the sqlcipher: https://github.com/sqlcipher/sqlcipher-android/pull/24/files. So we don't need to add these rules again. 
(Moreover, while building the release variant of the APK (sample-kotlin) I didn't notice any missing proguard rules.)